frontend/src/layout/navigation/TopBar/NotebookButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { IconNotebook } from '@posthog/icons'
+import { LemonButton } from '@posthog/lemon-ui'
+import { useActions } from 'kea'
+import { notebookPanelLogic } from 'scenes/notebooks/NotebookPanel/notebookPanelLogic'
+
+export function NotebookButton(): JSX.Element {
```

Comment:
Where does this component get referenced?

Reply:
Good catch! That's the result of a merge conflict and shouldn't be here.